graph-kiota/kiota.md

```diff
@@ -0,0 +1,180 @@
+### What is Kiota? 
+Learn [Kiota](https://github.com/microsoft/kiota#readme)
+
+### APIs that can use Kiota 
+- Any HTTP API with an Open API specification (For generators). Not limited to Microsoft Graph.
+
+### Kiota libraries
+
+Common for all languagues:
+- Kiota generator: Generates the request builders
+The generated code contains the APIClient class and the request builders. 
+
+```
+class APIClient {
+    httpCore : HttpCore;
+
+    getResourceById(): ResourceRequestBuilder;
+}
+```
+
+Libraries for each language:
+
+- Kiota abstractions
+    - This library contains only abstract classes and interfaces which can be implemented to connect the generated request builders with a client library that makes the call to the API.
+    [Kiota abstractions dotnet](https://github.com/microsoft/kiota/tree/main/abstractions/dotnet)
+    [Kiota abstractions java](https://github.com/microsoft/kiota/tree/main/abstractions/java)
+    [Kiota abstractions typescript](https://github.com/microsoft/kiota/tree/main/abstractions/typescript)
+
+    - Contains the following:
+        - HttpCore interface
+
+        ```
```

Comment:
again, let's refer to https://github.com/microsoft/kiota/blob/main/docs/kiotaabstractions.md#http-core instead of duplicating this.

Reply:
Created this here to capture the description and use case of each send function? 

Can you please provide the use cases for each of the send request, especially for `sendNoResponseContentAsync`? 

We can merge the documentations after we have captured information. 

We should update the code documentation as well.

Reply:
https://github.com/microsoft/kiota/blob/afc5e21da5c52a0c7fc683c4bb7c91825cdfffcd/abstractions/typescript/src/httpCore.ts#L41